src/util/utils.ts

```diff
@@ -221,18 +221,18 @@ export function fixFiatCurrencyCode(currencyCode: string) {
 
 // multiplier / exchange rate / ( 1 / unit )
 // 100000000 / $16500 / (1/$0.001) = ~6 sats
-export const calculateSpamThreshold = (rate: string, denom: EdgeDenomination) => {
-  return div(div(denom.multiplier, rate), '1000')
+export const calculateSpamThreshold = (rate: number, denom: EdgeDenomination) => {
+  return div(div(denom.multiplier, rate.toString()), '1000')
```

Comment:
Could we do `(parseFloat(multiplier) / rate) / 1000`? Of course, then you end up returning `number` instead of `string`, so the changes would ripple out further into the spam calculation, so feel free to skip if too complicated.

Reply:
I purposely didn't make this change because I wanted to limit the change to exchange rates and this function returns a crypto amount